internal/configs/configmaps.go

```diff
@@ -499,6 +499,15 @@ func ParseConfigMap(cfgm *v1.ConfigMap, nginxPlus bool, hasAppProtect bool, hasA
 				glog.Error("ConfigMap Key 'app-protect-physical-memory-thresholds' must follow pattern: 'high=<0 - 100> low=<0 - 100>'. Ignoring.")
 			}
 		}
+		if appProtectReconnectPeriod, exists := cfgm.Data["app-protect-reconnect-period-seconds"]; exists {
+			if period, err := ParseFloat64(appProtectReconnectPeriod); err == nil {
```

Comment:
Is a Float really needed? I would assume that seconds are int, float seems a little bit overkill 🤷‍♂️

Reply:
Hi. according to docs "Value type: number with decimal fraction" (https://docs.nginx.com/nginx-app-protect/configuration-guide/configuration/)